contracts/english_auction_rev1.1.scilla

```diff
@@ -947,6 +995,26 @@ transition End(
         (* payment tokens for the seller *)
         AddToPaymentTokenBalance seller payment_token_address seller_profit;
 
+        (* Paying out a commission if there is one *)
+        match is_commission_fee with
+          | True =>
+            AddToPaymentTokenBalance brand_recipient payment_token_address brand_commission_fee;
+
+            msg_to_collection_contract = {
+              _tag: "TokenSaleCallback";
+              _recipient: collection_contract;  (* TODO: get this address from somewhere *)
```

Comment:
@sam-zilliqa shouldn't this be  `collections_contract` or at least `collection_address`? If `collection_contract` is used here, it will be wrong once `UpdateCollectionContract` is called since it is `collection_address` that is updated.

Reply:
You are right, nicely spotted - I'll update it to collections_contract. This value is fetched[ from this line](https://github.com/Zilliqa/marketplace-contracts/blob/3b62f7065b776cbd12129c1fcfd5a40e0bb063b6/contracts/fixed_price_rev1.1.scilla#L473) which always reads the collection_address, so even if the contract address is updated at some point, that update will be reflected here.